VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -68,6 +71,22 @@ public void setEnd(String end) {
         this.end = end;
     }
 
+    public String getStart_display() {
+        return start_display;
+    }
+
+    public void setStart_display(String start_display) {
```

Comment:
Probably nicer to do these camelCase. Once we get past the API abstraction barrier we generally try to follow the local naming styles, but we haven't been totally consistent about it. But once it's getting to the level of a method name, it looks too weird otherwise.